resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,70 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*Id" : "r",
```

Comment:
You also have a number of places where you redundantly set an Id field explicitly to read-only. If you search for ```id" : "r"```, you should find most of them. Just another reason why I'm not loving this wildcard.

These comments apply to both admin and user auth. I won't duplicate my comments in user-auth.